conf/defaults.ini

```diff
@@ -1149,6 +1149,9 @@ commandPalette = true
 # Use dynamic labels in CloudWatch datasource
 cloudWatchDynamicLabels = false
 
+# Use HTTP 207 Multi Status for data source query responses
+datasourceQueryMultiStatus = false
```

Comment:
I don't think default-false feature flags need to be defined in here, right? By default feature flags are already off.

Reply:
Ah good shout. Just blindly copy pasta 👀